shiksha-website/shiksha-frontend/src/app/view/user/question-bank/question-bank-view/question-bank-view.component.html

```
@@ -83,64 +83,78 @@ <h2 class="text-center text-sm mt-1 font-bold text-content">
-          <div *ngIf="questionBankData.type !== 'Match the following'">
+          <div *ngIf="questionBankData.type !== 'MATCHING'">
             <div *ngFor="let questionData of questionBankData.questions;
             let i = index
           ">
               <div class="flex justify-between items-start mt-2 gap-2">
                 <p class="text-content">
-                  {{ i + 1 }}. {{ questionData.question | mathFormatting }}
+                  {{ i + 1 }}.
+                  <ng-container *ngFor="let item of contentItems(questionData.question)">
+                    <span *ngIf="isTextContent(item)">{{ item.content | mathFormatting }}</span>
+                    <img *ngIf="!isTextContent(item)" [src]="mediaSrc(item)" class="mt-2 max-h-64 max-w-full" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)